frontend/src/pages/Landing/LandingPage.tsx

```diff
@@ -172,11 +165,11 @@ export const LandingPage = (): JSX.Element => {
         >
           <Flex flexDir="column" flex={1}>
             <SectionTitleText>
-              No onboarding, no fees, and no code required
+              No onboarding, fees nor code required
```

Comment:
... no more oxford coma ? 😅 😜

Reply:
hahahah actually fine either way but I have never used it for copy bc usually trying to keep it as short as possible